code/addons/test/src/vitest-plugin/setup-file.ts

```diff
@@ -35,11 +35,10 @@ export const modifyErrorMessage = ({ task }: { task: Task }) => {
   }
 };
 
-// Enable this in 9.0
-// beforeAll(() => {
-//   if (globalThis.globalProjectAnnotations) {
-//     return globalThis.globalProjectAnnotations.beforeAll();
-//   }
-// });
+beforeAll(() => {
+  if (globalThis.globalProjectAnnotations) {
+    return globalThis.globalProjectAnnotations.beforeAll();
+  }
+});
```

Comment:
Do you want to pass down the context (test suite metadata)?
```suggestion
beforeAll((suite) => {
  if (globalThis.globalProjectAnnotations) {
    return globalThis.globalProjectAnnotations.beforeAll(suite);
  }
});
```

Reply:
storybook's beforeAll doesn't accepts it